onchain/rollups/test/foundry/dapp/helper/update-proofs.sh

```diff
@@ -52,22 +41,11 @@ runmachine() {
     docker run --rm \
         --name gen-proofs \
         -v "`pwd`/gen-proofs.sh:/opt/gen-proofs/gen-proofs.sh" \
+        -v "`pwd`/input:/opt/gen-proofs/input" \
         -v "`pwd`/output:/opt/gen-proofs/output" \
         -w /opt/gen-proofs \
         cartesi/server-manager-gen-proofs:devel \
-        ./gen-proofs.sh
-}
-
-# Decode strings in epoch status from Base64 to hexadecimal
-b64to16() {
-    python3 -m b64to16 output/finish_epoch_response_64.json > finish_epoch_response.json
-}
-
-# Generate Solidity library with proofs from epoch status
-genlib() {
-    pushd ../../../../../ >/dev/null
-    npx ts-node rollups/test/foundry/dapp/helper/genProofLibrary.ts
-    popd >/dev/null
+        ./gen-proofs.sh "$@"
```

Comment:
now that the `--setup` flag is removed, we could also remove this `$@`, although it could also stay

Reply:
Yes, we could remove it now, but I've left it in case we add any parameters later.

Reply:
Nevermind, I will remove it. :-)